### PR TITLE
Revert all Scaffold CommandSet Upgrade changes

### LINKED ISSUE #215: Patch104p breaks Skirmish Maps
**Repro:**
Load Skirmish map such as
https://github.com/xezon/GeneralsGamePatch/tree/main/Maps/SkirmishOnly/SkirmishTestPromotionsAIR

**Result Patch104p:**
Skirmish map loads and player cannot build.

**Result Original:**
Skirmish map loads and player can build normally.

Maybe related to:
https://github.com/xezon/GeneralsGamePatch/commit/c75622fa05bfaa28cac4a866ad1853f3d5107ca6

### PR DESCRIPTION
**Pull request by xezon**
Fixes #215

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/SuperWeaponGeneral.ini [hunk 1]
```diff
@@ -17488,7 +17357,7 @@ Object SupW_AmericaPowerPlant
   ; @bugfix - hanfield
   ; Added new default armorset that cannot be targeted or suffer SUBDUAL damage.
   ; This resolves the Microwave Tank exploit but does not prevent being properly disabled later.
-  ; 24/08/2021
+  ; 23/08/2021
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/SuperWeaponGeneral.ini:17360]: Why this?

File: Patch104pZH/GameFilesEdited/Data/INI/Object/SuperWeaponGeneral.ini [hunk 2]
```diff
@@ -17569,7 +17438,7 @@ Object SupW_AmericaPowerPlant
   ; MicrowaveTankFix01 grants a dummy object-class upgrade once the building is complete.
   ; MicrowaveTankFix02 upgrades the armor with said upgrade and allows subdual once again.
   ;
-  ; 24/08/2021
+  ; 23/08/2021
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/SuperWeaponGeneral.ini:17441]: And this?

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/SuperWeaponGeneral.ini:17441]: This was a mistake from hanfield which he submitted along with one of the scaffold changes. It's actually correct to revert this.

File: Patch104pZH/GameFilesEdited/Data/INI/Object/SuperWeaponGeneral.ini (hunk 1 above)
Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/SuperWeaponGeneral.ini:17360]: This was a mistake from hanfield which he submitted along with one of the scaffold changes. It's actually correct to revert this.

**Review by commy2 — APPROVED** [review]
Please squash before merging. Will help re-reverting the fix later if wanted too.

**Review by alanblack166 — APPROVED** [review]

**Pull request merged**